ecs-cli/modules/cli/configure/configure.go

```diff
@@ -14,40 +14,63 @@
 package configure
 
 import (
+	"errors"
+
 	"github.com/Sirupsen/logrus"
 	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/commands"
 	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/config"
 	"github.com/urfave/cli"
 )
 
-// ConfigureCluster is the callback for ConfigureCommand (cluster).
-func ConfigureCluster(context *cli.Context) {
-	// get relevant fields
+func fieldEmpty(flag string, context *cli.Context) bool {
+	field := context.String(flag)
+	if field == "" {
+		logrus.Errorf("%s can not be empty.", flag)
+		return true
+	}
+	return false
+}
+
+// Cluster is the callback for ConfigureCommand (cluster).
+// Note: The error returned is only used in the unit tests
+// This function is called by urfave/cli which does not check the error returned
+func Cluster(context *cli.Context) error {
+	// validate fields not empty
+	if fieldEmpty(command.RegionFlag, context) || fieldEmpty(command.ConfigNameFlag, context) || fieldEmpty(command.ClusterFlag, context) {
+		// fieldEmpty() will log the error; returned error is only used for unit tests
```

Comment:
Can you move this comment to be the description for `fieldEmpty` function?
And, I think we should just return the error that we are logging.